.travis.yml

```diff
@@ -1,67 +1,71 @@
-git blanguage: python
-sudo: required
-# if the https://travis-ci.org/ODM2/ODM2PythonAPI/requests ever says: missing config
-# validate at: http://lint.travis-ci.org/
-python:
-#  - "2.6"
-  - "2.7"
-#  - "3.2"
-#  - "3.3"
-#  - "3.4"
-#  - "3.5"
-#  - "3.5-dev" # 3.5 development branch
-#  - "nightly" # currently points to 3.6-dev
-# command to install dependencies
-cache:
-  directories:
-    - $HOME/.cache/pip/wheels
-    - $HOME/virtualenv/python2.7.9
+language: python
+
+sudo: false
+
 services:
   - mysql
   - postgresql
+
 addons:
   apt:
     packages:
-      - cmake
-      - unixodbc
-      - unixodbc-dev
+      # FIXME: when activating the services above I don't think these are needed.
       - odbcinst1debian2
       - odbcinst
-      - freetds-dev
-      - freetds-bin
       - tdsodbc
-      - libc6
       - e2fsprogs
+      # FIXME: the dependencies below can probably be replaced by conda.
+      - libc6
+      - freetds-dev
+      - freetds-bin
+      - unixodbc
+      - unixodbc-dev
       - mysql-client
+      # FIXME: I don't see the need for these anywhere.
```

Comment:
Which packages? If it's libproj-dev, libgeos-dev and libspatialiate-dev, I agree, they're not needed. I can't comment on `cmake`.